main/src/library/MutList.flix

```diff
@@ -54,25 +44,25 @@ namespace MutList {
     ///
     /// Returns a new empty mutable list with a default capacity.
     ///
-    pub def new(): MutList[a] & Impure = MutList(ref [$DEFAULT$; minCapacity()], ref 0)
+    pub def new(r: Region[r]): MutList[a, r] \ Write(r) =
+        MutList(ref [$DEFAULT$; minCapacity()] @ r @ r, ref 0 @ r)
```

Comment:
I understand this, but it looks crazy :O.

Maybe we should add least add parens? `ref ($DEFAULT$; minCapacity()] @ r) @ r` right?